### PR TITLE
Add Arxan Aarch64

### PR DESCRIPTION
**Pull request by enovella**
```sh
>  apkid ../../com.boost.care_2019-06-04.apk
[+] APKiD 2.1.0 :: from RedNaga :: rednaga.io
[*] ../../com.boost.care_2019-06-04.apk!classes2.dex
 |-> anti_debug : Debug.isDebuggerConnected() check
 |-> anti_vm : possible Build.SERIAL check
 |-> compiler : unknown (please file detection issue!)
 |-> obfuscator : Arxan (b - unconfirmed), unreadable field names, unreadable method names
[*] ../../com.boost.care_2019-06-04.apk!classes.dex
 |-> anti_vm : possible VM check
 |-> compiler : unknown (please file detection issue!)
 |-> obfuscator : Arxan (b - unconfirmed), unreadable field names, unreadable method names
[*] ../../com.boost.care_2019-06-04.apk!lib/armeabi-v7a/libUtilities.so
 |-> obfuscator : Arxan
[*] ../../com.boost.care_2019-06-04.apk!lib/arm64-v8a/libUtilities.so
 |-> obfuscator : Arxan
```

### CODE REVIEW

**Comment by enovella** [comment]
This assembly snippet is the way that Arxan breaks the functions in basic blocks' sets making the static reverse engineering task very hard to follow. This is a updated version of the previous Arxan 32bits rule.

**Comment by enovella** [comment]
ETA merge?

**Comment by CalebFenton** [comment]
LGTM, I can add what you said as a comment
@strazzere any thoughts?

**Comment by strazzere** [comment]
Seems sane to me, though I haven't tested it.

It seems unlikely that any sane developer would xor the address immediately before jumping to it for any reason other than obfuscation.
